recipe/meta.yaml

```diff
@@ -59,12 +59,14 @@ outputs:
         - natsort
         - nc-time-axis
         - netCDF4
+        - numba
         - numpy !=1.24.3  # severe masking bug
         - openpyxl
         - packaging
         - pandas
         - progressbar2
         - prov
+        - psyplot
```

Comment:
This also appears in l.73, please remove one of them

Reply:
Fixed in [5a4328f](https://github.com/conda-forge/esmvaltool-suite-feedstock/pull/24/commits/5a4328fc706299b9c0321a34ed324c49578da8b3)